src/cpu/flexcpu/notes.md

```diff
@@ -0,0 +1,45 @@
+
```

Comment:
Add documentation and call README
- How thread/cpu interact
- How thread control flow works
- How resources work
- Hints on configging